From 9991eb0f5eaf38aa7f0c61206937d4cd013e24c8 Mon Sep 17 00:00:00 2001
From: Jaime Torres Amate <jtamate@gmail.com>
Date: Mon, 10 Sep 2018 16:45:45 +0200
Subject: [PATCH] Don't assign twice the same key to the action New Tab

Summary:
QKeySequence::AddTab = Qt::CTRL + Qt::Key_T.
Leaving only the first avoids the warning window "There are two actions (New Tab, New Tab) that want to use the same shortcut (Ctrl+T)"

BUG: 398324

Test Plan:
su - test
unset KDE_FULL_SESSION
dolphin
After: No more the warning window.

Reviewers: #dolphin, elvisangelaccio

Reviewed By: #dolphin, elvisangelaccio

Subscribers: elvisangelaccio, kfm-devel

Tags: #dolphin

Differential Revision: https://phabricator.kde.org/D15371
---
 src/dolphinmainwindow.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dolphinmainwindow.cpp b/src/dolphinmainwindow.cpp
index f8b35d4a7..a862ae803 100644
--- a/src/dolphinmainwindow.cpp
+++ b/src/dolphinmainwindow.cpp
@@ -1079,7 +1079,7 @@ void DolphinMainWindow::setupActions()
     QAction* newTab = actionCollection()->addAction(QStringLiteral("new_tab"));
     newTab->setIcon(QIcon::fromTheme(QStringLiteral("tab-new")));
     newTab->setText(i18nc("@action:inmenu File", "New Tab"));
-    actionCollection()->setDefaultShortcuts(newTab, {Qt::CTRL + Qt::Key_T, QKeySequence::AddTab});
+    actionCollection()->setDefaultShortcuts(newTab, {QKeySequence::AddTab});
     connect(newTab, &QAction::triggered, this, static_cast<void(DolphinMainWindow::*)()>(&DolphinMainWindow::openNewActivatedTab));
 
     QAction* closeTab = KStandardAction::close(
